src/platform/ios/mod.rs

```diff
@@ -150,6 +150,23 @@ impl MonitorId {
     pub fn get_hidpi_factor(&self) -> f32 {
         1.0
     }
+
+    pub fn get_physical_extents(&self) -> Option<(u64, u64)> {
+        unsafe {
+            if setjmp(mem::transmute(&mut jmpbuf)) != 0 {
+                let app: id = msg_send![Class::get("UIApplication")?, sharedApplication];
+                let delegate: id = msg_send![app, delegate];
+                let state: *mut c_void = *(&*delegate).get_ivar("glutinState");
+                let state = state as *mut DelegateState;
```

Comment:
Okay, I've had a chance to work on iOS now. I don't believe this needs to be wrapped in `setjmp`, and you can just use `self.delegate_state` instead of re-retrieving everything.